packages/frontend/core/src/desktop/dialogs/setting/general-setting/plans/layout.tsx

```diff
@@ -75,35 +69,12 @@ export const PricingCollapsible = ({
 export interface PlanLayoutProps {
   cloud?: ReactNode;
   ai?: ReactNode;
-  cloudTip?: boolean;
+  scrollAnchor?: string;
 }
 
-export const PlanLayout = ({ cloud, ai, cloudTip }: PlanLayoutProps) => {
+export const PlanLayout = ({ cloud, ai, scrollAnchor }: PlanLayoutProps) => {
   const t = useI18n();
-  const [modal, setOpenSettingModal] = useAtom(openSettingModalAtom);
```

Comment:
the atom is removed, `scrollAnchor` is passed via props, and cloudTip is removed